lua/nvim-lsp-installer/servers/esbonio/init.lua

```diff
@@ -0,0 +1,15 @@
+local server = require "nvim-lsp-installer.server"
+local pip3 = require "nvim-lsp-installer.installers.pip3"
+
+return function(name, root_dir)
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        languages = { "rst" },
```

Comment:
I wonder if this should be the following instead? Reason being it seems like this server builds upon Sphinx specifically? Might be a bit confusing if you try to use this server for non-Sphinkx rst files?

```suggestion
    languages = { 'sphinx' },
```

Reply:
Sphinx is a superset of rst. Emm, and it seems vim has no filetype about sphinx, just rst.

![图片](https://user-images.githubusercontent.com/42114817/141104135-85561861-eed7-48df-830d-4ca75fbc3eb2.png)

Reply:
nvim/filetype.vim has no filetype sphinx:

![图片](https://user-images.githubusercontent.com/42114817/141104913-6466ee04-ee49-45f4-ac25-6d58b07b9a47.png)
![图片](https://user-images.githubusercontent.com/42114817/141104941-0e898b32-00d8-4986-a2a9-928dcada72bc.png)

But only have type rst:

![图片](https://user-images.githubusercontent.com/42114817/141105015-528d4b08-ec62-4294-97a5-ec716bc61536.png)

Reply:
The `language` field is actually only used to make these servers more discoverable, it has nothing to do with filetypes (this is governed by lspconfig). So an example of this is that by having `rst` listed as a language here, users can also install it with `:LspInstall rst`.

Reply:
So I was just thinking perhaps it's best to use sphynx as the target language as the server revolves around it

Reply:
OK，thanks for your notice